test/plenario_web/controllers/web/page_controller_test.exs

```diff
@@ -29,131 +29,158 @@ defmodule PlenarioWeb.Web.Testing.PageControllerTest do
-          [{-30, -10}, {-40, -40}, {-20, -40}, {-10, -20}, {-30, -10}]
-        ],
-        srid: 4326
-      }
-    {_, lower} = NaiveDateTime.from_iso8601("2015-01-01T00:00:00.0")
-    {_, upper} = NaiveDateTime.from_iso8601("2016-12-31T00:00:00.0")
-
-    {:ok, _} = MetaActions.update_bbox(meta3, bbox)
-    {:ok, _} = MetaActions.update_time_range(meta3, Plenario.TsRange.new(lower, upper))
-
-    {:ok, _} = MetaActions.update_bbox(meta4, bbox)
-    {:ok, _} = MetaActions.update_time_range(meta4, Plenario.TsRange.new(lower, upper))
-
-    m = MetaActions.get(meta1.id)
-    {:ok, _} = MetaActions.submit_for_approval(m)
-    m = MetaActions.get(meta1.id)
-    {:ok, _} = MetaActions.approve(m)
-    m = MetaActions.get(meta1.id)
-    {:ok, _} = MetaActions.mark_first_import(m)
-
-    m = MetaActions.get(meta2.id)
-    {:ok, _} = MetaActions.submit_for_approval(m)
-    m = MetaActions.get(meta2.id)
-    {:ok, _} = MetaActions.approve(m)
-    m = MetaActions.get(meta2.id)
-    {:ok, _} = MetaActions.mark_first_import(m)
-
-    m = MetaActions.get(meta3.id)
-    {:ok, _} = MetaActions.submit_for_approval(m)
-    m = MetaActions.get(meta3.id)
-    {:ok, _} = MetaActions.approve(m)
-    m = MetaActions.get(meta3.id)
-    {:ok, _} = MetaActions.mark_first_import(m)
-
-    m = MetaActions.get(meta4.id)
-    {:ok, _} = MetaActions.submit_for_approval(m)
-    m = MetaActions.get(meta4.id)
-    {:ok, _} = MetaActions.approve(m)
-    m = MetaActions.get(meta4.id)
-    {:ok, _} = MetaActions.mark_first_import(m)
-
-    params = %{
-      "starting_on" => "2014-11-01",
-      "ending_on" => "2015-11-01",
-      "coords" => "[[-30, -20], [-40, -50], [-10, -30], [-30, -20]]",
-      "zoom" => 10
-    }
-    response =
-      conn
-      |> get(page_path(conn, :explorer, params))
-      |> html_response(:ok)
-
-    refute response =~ meta1.name
-    refute response =~ meta2.name
-    assert response =~ meta3.name
-    assert response =~ meta4.name
-
-    params = %{
-      "starting_on" => "2016-11-01",
-      "ending_on" => "2017-11-01",
-      "coords" => "[[30, 20], [40, 50], [10, 30], [30, 20]]",
-      "zoom" => 10
-    }
-    response =
-      conn
-      |> get(page_path(conn, :explorer, params))
-      |> html_response(:ok)
-
-    assert response =~ meta1.name
-    assert response =~ meta2.name
-    refute response =~ meta3.name
-    refute response =~ meta4.name
-
-    params = %{
-      "starting_on" => "2014-11-01",
-      "ending_on" => "2015-11-01",
-      "coords" => "[[30, 20], [40, 50], [10, 30], [30, 20]]",
-      "zoom" => 10
-    }
-    response =
-      conn
-      |> get(page_path(conn, :explorer, params))
-      |> html_response(:ok)
-
-    refute response =~ meta1.name
-    refute response =~ meta2.name
-    refute response =~ meta3.name
-    refute response =~ meta4.name
-    assert response =~ "Sorry, we didn't find any data sets that matched your criteria"
-  end
+  # @tag :anon
+  # test "search explorer", %{conn: conn} do
```

Comment:
Tomorrow I can give you a hand on trying to debug this. I'm a firm believer that "comment out the test" is always the wrong option, even if the alternative is "rewrite the test".

Reply:
Me too.